### DISCUSSION TITLE
how to check jsPsych version

### DISCUSSION THREAD

**Discussion by sonhkim** [jspsych/jsPsych#1626] [Q&A] [answered]
Hi folks, 

How can I check the version of currently installed jsPsych? I tried putting 

    var version = jsPsych.version();
    console.log(version);

in the experiment.html file, but the console throws the following error. 

Uncaught TypeError: jsPsych.version is not a function
    at experiment.html:14

Anyone has an idea? Thanks!

**Comment by jodeleeuw** [comment, marked as answer]
We added this feature in version 6.3.0, so previous versions will not return a value. 

You are using < 6.3 😬

**Reply by sonhkim** [reply]
Okay, at least then it's prior to 6.3! Any other way to get the exact version? Cognition.run wants me to specify it.

**Reply by jodeleeuw** [reply]
You could download the releases from github and see how many lines are in jspsych.js. I'm not certain, but I suspect that each version has a different line number count.
